src/scribe_data/utils.py

```diff
@@ -10,18 +10,23 @@
     get_language_from_iso,
```

Comment:
Comparing this PR with #89, it appears to me all changes here are already reflected in the other PR.

I am thinking of simply closing this one here and continuing on #89 directly - is that alright? Mostly to avoid making any changes here and then later having to redo/reconcile the same on the other PR.

CC @andrewtavis

Reply:
Yeah!, The other PR was checked from this this branch, merging that one makes sense.

Reply:
Great! Sounds good then - I'll do my review on that other PR :+1:

Reply:
Thanks both! 😊